rest/src/main/resources/scripts/ZStack SDK Go 开发规范与标准.md

```
@@ -0,0 +1,1096 @@
+
+### 安装
+
+```bash
+go get github.com/terraform-zstack-modules/zstack-sdk-go-v2
+```
+
+### 初始化客户端
+
+> **注意**: `DefaultZSConfig` 需要显式传入 context path (如 `"/zstack"`)。
+
+```go
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)